android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -400,6 +400,12 @@ public void removeRouteChangeListener() {
     }
   }
 
+  @ReactMethod
+  public void adjustStreamVolume(final int streamType, final int direction, final int flags) {
```

Comment:
Should the parameters be final?

Reply:
I just followed other methods in the same file. But, It is not needed to be final, when I check official document.